### DISCUSSION TITLE
Option to disable 'recent comments' widget.

### DISCUSSION THREAD

**Discussion by wjpvandenheuvel** [auroral-ui/hexo-theme-aurora#103] [Ideas]
I was wondering how to disable this widget. When starting a new blog and (obviously) nobody has commented yet, the recent comments widget remains empty and tries to load comments that are not there. 

Also, the amount of comments (66 in this case) still seems to correspond with the amount of comments on your own website. https://tridiamond.tech/

![image](https://user-images.githubusercontent.com/416365/118966607-21393000-b96a-11eb-8fe6-018f605fa7a2.png)

**Comment by deskside** [comment]
Me having the same issue!

**Comment by olina0421** [comment]
same here, anyone knows how to remove it? Thanks!

**Reply by deskside** [reply]
@wjpvandenheuvel  @olina0421 Hi there, I have asked the author and he replied that it can't be turned off.
